CONTRIBUTING.md

```diff
@@ -56,6 +56,25 @@ release branch. For example, if the most recent branch is `release-0.2`, and the
 development for v0.3.0, a bug fix that merged to `master` that also affects `v0.2.x` may be considered for backporting
 to `release-0.2`. We generally do not accept PRs against older release branches.
 
+### Proposal process (CAEP)
+
+The Cluster API Enhacement Proposal is the process this project uses to adopt new features, or changes to the APIs.
+
+- The template, and accepted proposals live under `docs/proposals`.
+- A proposal SHOULD be introduced and discussed during the weekly community meetings.
+- A proposal MUST be submitted first to the community using a collaborative writing platform, preferably Google Docs.
```

Comment:
SHOULD (event if recommended)?

Reply:
We want to avoid straight-to-PR proposals, collaborative environments like Google Docs are probably the best way to kickoff discussions and collaboration

Reply:
+1 to looser language here (like SHOULD), we could always redirect a straight-to-PR situation to a collaborative doc, and this would allow relatively trivial proposals a bit more of a streamlined process.